include/openssl/rsa.h

```diff
@@ -45,6 +45,10 @@ extern "C" {
 # define RSA_3   0x3L
 # define RSA_F4  0x10001L
 
+/* based on RFC 8017 appendix A.1.2 */
+# define RSA_ASN1_VERSION_DEFAULT 0
+# define RSA_ASN1_VERSION_MULTI 1
```

Comment:
It would be nice if you aligned the numbers...  something like this:

``` C
# define RSA_ASN1_VERSION_DEFAULT       0
# define RSA_ASN1_VERSION_MULTI         1
```

Reply:
Sure